org/mozilla/jss/pkcs11/PK11Signature.c

```diff
@@ -644,11 +823,17 @@ Java_org_mozilla_jss_pkcs11_SigContextProxy_releaseNativeResources
 
     /* Free the context and the proxy */
     if(proxy->type == SGN_CONTEXT) {
-        SGN_DestroyContext( (SGNContext*)proxy->ctxt, PR_TRUE /*freeit*/);
-    } else {
+        SGN_DestroyContext( (SGNContext*)proxy->ctxt, PR_TRUE /*freeit*/); } else {
         PR_ASSERT(proxy->type == VFY_CONTEXT);
         VFY_DestroyContext( (VFYContext*)proxy->ctxt, PR_TRUE /*freeit*/);
     }
+
+    if(proxy->arena != NULL) {
+        PR_fprintf(PR_STDERR,"About to remove arena!\n");
```

Comment:
Debug statement should probably be removed.

Reply:
And space on the line above too :-)

Reply:
Done.